dev-guide/cluster-version-operator/dev/clusteroperator.md

```diff
@@ -207,3 +207,57 @@ Conditions determine when the CVO considers certain actions complete, the follow
 [2] Upgrade will not proceed with upgrading components in the next runlevel until the previous runlevel completes.
 
 See also: https://github.com/openshift/cluster-version-operator/blob/a5f5007c17cc14281c558ea363518dcc5b6675c7/pkg/cvo/internal/operatorstatus.go#L176-L189
+
+### Related Objects
+
+Related objects are the fundamental set of objects related to your operator.
+As mentioned previously, the insights operator and must-gather depend on related images to collect relevant diagnostic information about each ClusterOperator.
+
+In addition to specifying related objects statically in your operator's manifests, you should also implement logic in your operator to keep related objects up-to-date.
```

Comment:
Do we know whether there's anything in library-go that could be used here?

Reply:
Look like there's a `StatusSyncer` https://github.com/openshift/library-go/blob/3ea4b775d418ad87858610b96ed8ba9bf22137a1/pkg/operator/status/status_controller.go#L59 in library-go that has some facilities for relatedObjects. But mentioning that library may make more sense for a follow-up since it seems to cover helpers for the entire status object, not just relatedObjects.